aDataGenerator/dataGenerator.js

```diff
@@ -0,0 +1,225 @@
+  "Rogan", "Codie", "Maisum", "Obosa", "Shazil", "Lawrie", "Aleksandar", "Jae", "Bertie",
+  "Killian", "Suraj", "Joss", "Kerr", "Miguel", "Albie", "Pietro", "Kruz", "Ceejay", "Zuriel", 
+  "Lachlann", "Teagan", "Deegan", "Zachary-Marc", "Madison-Jake", "Alvin", "Rorie",
+  "Carson", "Thiago", "Dyllan", "Russell", "Kurtis-Jae", "Magnus", "Rudi", "Tian",
+  "Aryian", "Immanuel", "Jacki", "Sorley", "Roger", "Tayye", "Fares", "Abir", "Solomon",
+  "Tisloh", "Ikemsinachukwu", "Israel", "Ayren", "Leighton", "Craig-James", "Latif",
+  "Tiernan", "Marco", "Conner", "Zhong", "Lucas", "Allesandro", "Rowen", "Coby", "Fox",
+  "Luc", "Zhong", "Pearsen", "Roger", "Selasi", "Sohaib", "Dolan", "Torquil", "Thrinei",
+  "Jonathan", "Kaidyn", "Amolpreet", "Jameil", "Buddy", "Conlyn", "Corey-James",
+];
+
+const houseType = [
+  'ENTIRE CABIN', 'ENTIRE HOUSE', 'ENTIRE APARTMENT', 'PRIVATE ROOM', 'ENTIRE LOFT',
+  'CAMPER', 'EARTH HOUSE', 'DOME HOUSE', 'TREE HOUSE', 'ENTIRE CABIN', 'ENTIRE HOUSE',
+  'ENTIRE APARTMENT', 'PRIVATE ROOM', 'ENTIRE CABIN', 'ENTIRE HOUSE', 'ENTIRE APARTMENT',
+];
+
+const cities = [
+  "Fort Wayne", "Pomona", "Racine", "Shreveport", "Corpus Christi", "Tulsa", "San Antonio",
+  "Oklahoma City", "Santa Ana", "Thornton", "Fitchburg", "Vallejo", "Barnstable", "Lansing",
+  "Bellevue", "Stockton", "Erie", "Sebastian", "Memphis", "Vero Beach", "Fort Lauderdale", "Layton",
+  "Myrtle Beach", "Daly City", "Port St. Lucie", "Albany", "Atlanta", "Newburgh", "Huntington", "Columbia",
+  "Moreno Valley", "Akron", "Fontana", "Riverside", "Grayslake", "Orlando", "Corpus Christi", "Lewisville",
+  "Fullerton", "Inglewood", "Honolulu", "Tampa", "Arlington", "Coral Springs", "Springfield", "Nashville",
+  "Monterey", "Hollywood", "Springfield", "Howell", "Elk Grove", "Lakeland", "Round Lake Beach", "Durham",
+  "Baltimore", "Saginaw", "Denton", "Bellevue", "Macon", "Shreveport", "Orlando", "Howell", "Cape Coral",
+  "Madison", "St. Paul", "Tyler", "Temecula", "Merced", "Tucson", "Virginia Beach", "Kenosha",
+  "Westminster", "Havre de Grace", "Apple Valley", "Fargo", "High Point", "Nashville", "Brighton",
+  "St. Paul", "Las Cruces", "Champaign", "Fayetteville", "Waterloo", "Santa Clara", "Macon", "Olympia",
+  "Knoxville", "Visalia", "Waterbury", "Beaumont", "Bremerton", "Port Orange", "Garland", "Bonita Springs",
+  "Providence", "Des Moines", "Omaha", "North Port", "Clarksville", "Norfolk",
+];
+
+const highlights = ['Sparkling clean', 'Great Location', 'Great check-in experience'];
+
+const optionalDescriptionSections = ['Guess access', 'Interaction with guests', 'Other things to note'];
+
+const amenitiesSections = ['Family features', 'Dining', 'Facilities', 'Guess access', 'Logistics', 'Bed and bath',
+  'Safety features', 'Outdoor'];
+
+const randomInterger = (min, max) => Math.floor(Math.random() * ((max - min) + 1)) + min;
+
+const arbitrarySentence = (arrEntry, numOfWords, period) => {
```

Comment:
should use better name for variables that will be a boolean. Instead of "period", use something like "hasPeriod"